_commands/os_x/pbcopy.md

```diff
@@ -0,0 +1,55 @@
+---
+---
+
+pbcopy
+-------
+
+`pbcopy` is used to copy text from the terminal into the clipboard.
+
+~~~ bash
+#Copies the contents of a file into the clipboard
```

Comment:
In this instance are you referring to putting spaces between the '#' and the comment itself? like this:

```
# Copies the contents of a file into the clipboard
```